src/deps.deno.ts

```diff
@@ -1,11 +1,55 @@
-import { hmac } from "https://deno.land/x/hmac@v2.0.1/mod.ts";
+import { createHash, createHmac } from "https://deno.land/std@0.160.0/node/crypto.ts"
```

Comment:
This part of the standard library has been removed by now. Do you know how this would ideally be done today?

Reply:
I didn't even noticed this before o.o, so according to their docs (https://deno.com/blog/v1.30#support-for-built-in-nodejs-modules) we could just use `node:crypto` and that would work perfectly. However I'm not a fan of using node stuff in deno, so I'll take a closer look in the API to see if they have already ported the crypto module or not

Reply:
Awesome, thanks!

Reply:
AFAIK all crypto stuff in Deno happens with the Web Crypto API (https://developer.mozilla.org/en-US/docs/Web/API/Web_Crypto_API) but it's quite cumbersome to work with, so perhaps it would be great to have a wrapper around it? I'm not sure

Reply:
Are you still interested in fixing this up?